src/components/task-step/ends.cjsx

```diff
@@ -13,6 +13,8 @@ TaskStep = require './index'
 {TaskStepStore} = require '../../flux/task-step'
 {CardBody, PinnableFooter} = require '../pinned-header-footer-card/sections'
 
+ReactCSSTransitionGroup = React.addons.CSSTransitionGroup
```

Comment:
I'm a little worried about adding in CSS Transitions in some parts of the code but not others but Fred explicitly mentioned this.